Spigot/src/main/java/net/notfab/hubbasics/spigot/nms/CraftBukkitVersion.java

```diff
@@ -40,7 +40,9 @@
 
     v1_16_X("1.16.x", 1.16, "v1_16_R1", "v1_16_R2"),
```

Comment:
Please update this entry with all NMS versions.

`v1_16_X("1.16.x", 1.16, "v1_16_R1", "v1_16_R2", "v1_16_R3")`

Reply:
I've modified the requested line